composer.json

```diff
@@ -1,29 +1,31 @@
 {
     "name": "olcs/olcs-selfserve",
     "description": "OLCS Self Service Web Site",
-    "repositories": [
-        {
-            "packagist": false
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/OLCS",
-            "type": "composer"
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "type": "composer"
-        }
-    ],
     "require": {
         "php": ">=7.4",
         "dvsa/laminas-config-cloud-parameters": "^0.1.0",
+        "laminas/laminas-authentication": "^2.6",
+        "laminas/laminas-eventmanager": "^2.6",
+        "laminas/laminas-form": "^2.13",
+        "laminas/laminas-http": "^2.8",
+        "laminas/laminas-mvc": "^2.7",
+        "laminas/laminas-servicemanager": "^2.7",
+        "laminas/laminas-session": "^2.8",
+        "laminas/laminas-view": "^2.11",
+        "olcs/olcs-auth": "^5.0@alpha",
+        "olcs/olcs-common": "^4.206.516",
         "olcs/olcs-frontend-deps": "~4.206.0",
-        "olcs/olcs-laminas": "~2.5.0"
+        "olcs/olcs-logging": "^5.0@alpha",
```

Comment:
We don't need a @alpha prefix. Composer should pick the appropriate version.

Reply:
done